tests/utils/breadcrumbs.test.ts

```diff
@@ -0,0 +1,98 @@
+import { NextPageContext } from 'next';
+import breadcrumb from '../../src/utils/breadcrumbs';
+import {
+    getMockContext,
+    mockSingleAdultCsvUploadFromMatchingBreadcrumbs,
+    mockReturnAnyoneManualFromOutboundMatchingBreadcrumbs,
+    mockPeriodGeoZoneSeniorFromCsvZoneUploadBreadcrumbs,
+    mockFlatFareStudentFromDefinePassengerTypeBreadcrumbs,
+    mockMultiServicesAnyoneFromMultipleProductValidityPageBreadcrumbs,
+    mockMultiServicesAnyoneFromPeriodValidityPageBreadcrumbs,
+} from '../testData/mockData';
+
+describe('breadcrumbs', () => {
+    let ctx: NextPageContext;
+
+    it('creates the correct array of Breadcrumbs if user is on matching page having selected single, adult and csv upload', () => {
```

Comment:
Do we need to add a test if on home page?

Reply:
Now added.